core/src/test/scala/unit/kafka/controller/ControllerIntegrationTest.scala

```diff
@@ -43,34 +43,34 @@ class ControllerIntegrationTest extends ZooKeeperTestHarness {
   def testEmptyCluster(): Unit = {
     servers = makeServers(1)
     TestUtils.waitUntilTrue(() => zkUtils.pathExists(ZkUtils.ControllerPath), "failed to elect a controller")
-    TestUtils.waitUntilTrue(() => zkUtils.readData(ZkUtils.ControllerEpochPath)._1.toInt == KafkaController.InitialControllerEpoch,
+    TestUtils.waitUntilTrue(() => zkUtils.readDataMaybeNull(ZkUtils.ControllerEpochPath)._1.map(_.toInt) == Some(KafkaController.InitialControllerEpoch),
```

Comment:
Since we seem to do this in every test, can we add a method that takes the expected epoch called `waitUntilControllerEpoch` or something?